### DISCUSSION TITLE
Does this module support Menu export future ?

### DISCUSSION THREAD

**Discussion by syedrazaalino** [SnowdogApps/magento2-menu#252] [General]
I can see in code there is ImportExport but when installing this module, i can only see Import option, there is no way to export it.

**Comment by ofrankowska** [comment]
Hi! First of all, sorry for the later reply. You can find the export functionality here:

<img width="1696" alt="Screenshot 2023-08-04 at 19 12 50" src="https://github.com/SnowdogApps/magento2-menu/assets/49198312/f691af7f-fe8c-4cda-8db6-42ad154669e4">
